Harvest.Web/Controllers/Api/TicketController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Harvest.Core.Data;
+using Harvest.Core.Domain;
+using Harvest.Core.Models;
+using Harvest.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Harvest.Web.Controllers.Api
+{
+    public class TicketController : Controller
+    {
+        private readonly AppDbContext _dbContext;
+        private readonly IUserService _userService;
+
+        public TicketController(AppDbContext dbContext, IUserService userService)
+        {
+            this._dbContext = dbContext;
+            this._userService = userService;
+        }
+
+
+        // create a new request via react
+        [HttpGet]
+        public ActionResult Create()
+        {
+            return View("React");
+        }
+
+        [HttpPost]
+        [Authorize(Policy = AccessCodes.PrincipalInvestigator)]
+        public async Task<ActionResult> Create(int projectId, [FromBody] Ticket ticket)
+        {
+            if (projectId != ticket.ProjectId)
+            {
+                return BadRequest("Wrong Project Id");
+            }
+
+            var project = await _dbContext.Projects.SingleAsync(a => a.Id == projectId);

```

Comment:
why bother getting project?

Reply:
@srkirkland I'm returning to the project details

Reply:
wouldn't it make more sense to return the ticket details since it's to create a new ticket?

Reply:
I was copying code. I don't have any other ticket pages at the moment.